### ISSUE TITLE
New-HPOVLogicalInterconnectGroup with -Scope parameter

### ISSUE THREAD

**Issue by DungKHoang** [HewlettPackard/POSH-HPOneView#443]
When creating a new LIG, and use the -Scope parameter, The LIG is not added to the resource scope. 
OneView library 5.0

### Expected Behavior
LIG should be added to the scope

### Steps to reproduce

```PowerShell
$Scope1 			= New-HPOVScope -name 'Scope-LIG'
# -------------- Attributes for Logical Interconnect Group "LIG-FC"
$name                       = "LIG-FC"
$frameCount                 = 1
$bayConfig                  = @{
	Frame1 = @{
		Bay1 = 'SEVC32GbFC';
		Bay4 = 'SEVC32GbFC'
	}
}
$redundancyType             = "Redundant"
$fabricModuleType           = "SEVCFC"
$InterconnectBaySet         = 1
$networkLoopProtection      = $True
$Scope1                     = Get-HPOVScope -Name "Scope-LIG"

New-HPOVLogicalInterconnectGroup -Name $name -Bays $bayConfig -FabricModuleType $fabricModuleType -FrameCount $frameCount -InterConnectBaySet $InterconnectBaySet -enablenetworkLoopProtection:$networkLoopProtection -Scope $Scope1 #, $Scope2

```

### Version Information
See Log file attached
[2001170719_HPOV_transcript.log](https://github.com/HewlettPackard/POSH-HPOneView/files/4077481/2001170719_HPOV_transcript.log)

**Comment by ChrisLynchHPE** [comment]
Thanks for reporting this.  A workaround until this is fixed is to first create the resource, then use `Add-HPOVResourceToScope` Cmdlet.

**Comment by ChrisLynchHPE** [comment]
This was fixed in an updated 5.00 library.

Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
